task/net-api/src/lib.rs

```diff
@@ -54,7 +54,7 @@ pub enum PhyError {
 #[repr(u32)]
 pub enum KszError {
     /// This functionality is not available on the given board
-    NotAvailable,
+    NotAvailable = 1,
```

Comment:
Should you add this for documentation for each of the enums?:
enum Foo {
  _InterpretendAsOk = 0,
 ...

Reply:
John added a check to the `IdolError` macro!